server/src/main/java/org/elasticsearch/index/IndexMode.java

```diff
@@ -185,6 +192,13 @@ public IdFieldMapper buildIdFieldMapper(BooleanSupplier fieldDataEnabled) {
         public DocumentDimensions buildDocumentDimensions() {
             return new TimeSeriesIdFieldMapper.TimeSeriesIdBuilder();
         }
+
+        @Override
+        public IndexLongFieldRange getConfiguredTimestampRange(IndexMetadata indexMetadata) {
+            long min = indexMetadata.getTimeSeriesStart().toEpochMilli();
+            long max = indexMetadata.getTimeSeriesEnd().toEpochMilli();
+            return IndexLongFieldRange.NO_SHARDS.extendWithShardRange(0, 1, ShardLongFieldRange.of(min, max));
```

Comment:
Hey. So. Like. I think I made `getTimestampBound` and it does like nearly the same thing, right? I feel like that method should be involved in this PR somehow. Or maybe `IndexSettings.timestampBounds` should be.

Reply:
Good point. I think I can reuse `getTimestampBound(...)` and then `IndexMetadata#getTimeSeriesTimestampRange(...)` use that to build a `IndexLongFieldRange` instance.

Reply:
`TimestampBounds` needs `IndexScopedSettings` which I don't have in the context In want to use it.
This technically only needed for dynamically updating endTime field, which is my case isn't needed.
But maybe with a minor change to `TimestampBounds`,  I can use it too.

Reply:
Yes, I can do that.